aptos-move/e2e-tests/src/executor.rs

```diff
@@ -577,6 +577,7 @@ impl FakeExecutor {
         function_name: &str,
         type_params: Vec<TypeTag>,
         args: Vec<Vec<u8>>,
+        aggregator_enabled: bool,
```

Comment:
nit: I think you can hide the flag as a field in `FakeExecutor` to avoid passing it to exec everywhere (and default to true)? And then just have a setter.

Reply:
Done.